spec/Section 7 -- Response.md

```diff
@@ -64,18 +64,18 @@ If the `data` entry in the response is present (including if it is the value
 were raised during execution. If field errors were raised during execution, it
 should contain those errors.
 
-**Request errors**
+**Request Errors**
 
 Request errors are raised before execution begins. This may occur due to a parse
```

Comment:
This section still does not read well.

```
Request errors are raised before execution begins. This may occur due to a parse grammar or validation error in the request document, an inability to determine which operation to execute, or an invalid input value of variables.
```

Still does not read well but is a bit better.